Automattic/studio
# Studio: Update php-wasm ( playground dependecies)

Fixes https://github.com/Automattic/dotcom-forge/issues/8867

## Proposed Changes

This PR updates the following packages to the latest version:
- @php-wasm/node
- @php-wasm/universal
- @php-wasm/scopes
- @wp-playground/blueprints

During the update we ran into various issues, some of which required changes in the wordpress-playground repository:
- Fix issue with types definitions that were missing
- Fix issue with websocket proxy server not closing when a phpruntime is exited.

We had to change a couple of things in the current implementation of wp-now within Studio as it was relying on outdated APIs. These changes mainly relate to how we obtain a PHP instance.

## Testing Instructions



- Try running the app on MacOS and Windows
-  We should ideally systematically go over the list of smoke tests to ensure that this update did not break anything that was previously working.

## Pre-merge Checklist
